<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PopKrop</class>
 <widget class="QMainWindow" name="PopKrop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PopKrop</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="b_go">
        <property name="text">
         <string>Go!</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="e_url"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QProgressBar" name="pb_load">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="browserPlace"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
